$OpenBSD: patch-libavcodec_aacenc_pred_h,v 1.2 2015/10/17 06:49:22 ajacoutot Exp $

aacenc: shorten name of ff_aac_adjust_common_prediction

aacenc_pred: correct header information

--- libavcodec/aacenc_pred.h.orig	Mon Sep  7 21:58:01 2015
+++ libavcodec/aacenc_pred.h	Fri Oct 16 23:42:23 2015
@@ -21,7 +21,7 @@
 
 /**
  * @file
- * AAC encoder main prediction
+ * AAC encoder main-type prediction
  * @author Rostislav Pehlivanov ( atomnuker gmail com )
  */
 
@@ -40,7 +40,7 @@
 #define PRED_SFB_START 10
 
 void ff_aac_apply_main_pred(AACEncContext *s, SingleChannelElement *sce);
-void ff_aac_adjust_common_prediction(AACEncContext *s, ChannelElement *cpe);
+void ff_aac_adjust_common_pred(AACEncContext *s, ChannelElement *cpe);
 void ff_aac_search_for_pred(AACEncContext *s, SingleChannelElement *sce);
 void ff_aac_encode_main_pred(AACEncContext *s, SingleChannelElement *sce);
 
